src/deploymentfrequency.ps1

```diff
@@ -8,7 +8,7 @@ Param(
     [string] $actionsToken = "",
     [string] $appId = "",
     [string] $appInstallationId = "",
-    [string] $privateKey = ""
+    [string] $appPrivateKey = ""
```

Comment:
@eBerdnA FYI: Renamed the parameter to be consistent with the others - so that all of the GitHub app parameters have a prefix of 'app'